spec/features/visitor/sign_up_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature "Sign Up" do
+  let(:user_attributes) { attributes_for(:user).slice(:full_name, :email, :password, :password_confirmation) }
+  let(:registered_user) { User.find_by_email(user_attributes[:email]) }
+
+  background do
+    visit new_user_registration_path
+
+    fill_form(:user, user_attributes)
+
+    click_button "Sign up"
+  end
+
+  scenario "Visitor signs up" do
+    open_email(registered_user.email)
+
+    expect(current_email).to have_subject "Confirmation instructions"
+    expect(current_email).to have_body_text(registered_user.full_name)
+  end
+
+  scenario "Visitor confirms account" do
```

Comment:
I don't think that it's a good idea:

* we'll have to duplicate full sign up process (fill in, submit, open email).
* it's not separate features, it's the same scenario: user is not signed up until he/she _confirmed_ email address.

With above said, I combined both `scenario`s in https://github.com/fs/rails-base/commit/7a02ce0a451f6794ee280510332f81525810c099. So now we have complete feature: fill form, submit, get email, follow link, confirm account, done.